src/float_impl.rs

```diff
@@ -29,6 +29,18 @@ impl<F: Float, C: FloatChecker<F>> Clone for NoisyFloat<F, C> {
 
 impl<F: Float, C: FloatChecker<F>> Copy for NoisyFloat<F, C> {}
 
+impl<F: Float, C: FloatChecker<F>> AsRef<F> for NoisyFloat<F, C> {
+    fn as_ref(&self) -> &F {
+        &self.value
+    }
+}
+
+impl<F: Float, C: FloatChecker<F>> AsMut<F> for NoisyFloat<F, C> {
+    fn as_mut(&mut self) -> &mut F {
```

Comment:
This can be dangerous. Giving the user a mutable reference would allow them to change the value within NoisyFloat bypassing the float checker C.

Reply:
I should have noticed that. Thanks for catching it!